### PR TITLE
Add troubleshooting step for pam home dirs

### PR DESCRIPTION
**Pull request by Firstyear**

### CODE REVIEW

File: book/src/integrations/pam_and_nsswitch.md [hunk 1]
```diff
@@ -489,3 +489,10 @@ may have your laptop in a park without wifi.
 Clearing the cache, however, completely wipes all local data about all accounts and groups. If you
 are relying on this cached (but invalid) data, you may lose access to your accounts until other
 communication issues have been resolved.
+
+### Home directories are not created via SSH
+
+Ensure that `UsePAM yes` is set in `sshd_config`. Without this the pam session module won't be
+triggered which prevents the background task being completed.
+
+
```
Comment (yaleman) [inline book/src/integrations/pam_and_nsswitch.md:498]: The extra newlines here are making deno sad

**Review by yaleman — CHANGES_REQUESTED** [review]

File: book/src/integrations/pam_and_nsswitch.md (hunk 1 above)
Comment (Firstyear) [reply book/src/integrations/pam_and_nsswitch.md]: I have no idea how they were added, just deleted them.

File: book/src/integrations/pam_and_nsswitch.md [hunk 2]
```diff
@@ -489,3 +489,10 @@ may have your laptop in a park without wifi.
 Clearing the cache, however, completely wipes all local data about all accounts and groups. If you
 are relying on this cached (but invalid) data, you may lose access to your accounts until other
 communication issues have been resolved.
+
+### Home directories are not created via SSH
+
+Ensure that `UsePAM yes` is set in `sshd_config`. Without this the pam session module won't be
+triggered which prevents the background task being completed.
+
```
Comment (yaleman) [inline book/src/integrations/pam_and_nsswitch.md]:
```suggestion
```

**Review by yaleman — CHANGES_REQUESTED** [review]

File: book/src/integrations/pam_and_nsswitch.md (hunk 1 above)
Comment (yaleman) [reply book/src/integrations/pam_and_nsswitch.md]: yeah, easy to do :smile

**Review by yaleman — APPROVED** [review]

**Pull request merged**